src/Core/Dfe.Complete.Application/Common/Exceptions/NotDevOrTestEnvironmentException.cs

```diff
@@ -4,7 +4,7 @@
 namespace Dfe.Complete.Application.Common.Exceptions
 {
     [ExcludeFromCodeCoverage]
-    public class NotDevEnvironmentException() : Exception("Command is only available on Development environment")
+    public class NotDevOrTestEnvironmentException() : Exception("Command is only available on Development environment")
```

Comment:
Can you update the message to reflect the different environments?
```suggestion
    public class NotDevOrTestEnvironmentException() : Exception("Command is only available on Development and Test environments")
```